odk1-src/central-install-digital-ocean.rst

```diff
@@ -109,16 +127,11 @@ Once that is complete, congratulations! You have installed your copy of ODK Cent
 Starting up ODK Central
 -----------------------
 
-Now, we want to run your new ODK server software. But we don't want to only run it once: if we do that, then if your machine crashes or restarts, the software won't start back up. We want to tell the machine to always run the server. To teach it to do this, we have to copy a file to the right spot. To do this, run ``cp files/docker-compose@.service /etc/systemd/system``.
```

Comment:
I think it's important to have some notice that systemd is no longer being used. It might even be worth it to ask people to remove it if they want.

Reply:
i think it doesn't really do anything bad that it's there, and it isn't even bad if people continue to use the `systemctl` commands to manage the cluster.